test/integration/NotificationWorkItemRepoSpec.scala

```diff
@@ -213,5 +213,16 @@ class NotificationWorkItemRepoSpec extends UnitSpec
       await(repository.findById(item2.id)).get.status shouldBe Failed
       await(repository.findById(item3.id)).get.status shouldBe PermanentlyFailed
     }
+
```

Comment:
what about proving save new with status PermanentlyFailed?

Reply:
Added a new test that passes the status explicitly.